packages/react-experiments/src/components/UnifiedPicker/hooks/useFloatingSuggestionItems.ts

```diff
@@ -1,10 +1,19 @@
 import * as React from 'react';
+import { ISuggestionsHeaderFooterProps } from '@fluentui/react/lib/FloatingPicker';
 
 export interface IUseFloatingSuggestionItems<T> {
   focusItemIndex: number;
   setFocusItemIndex: React.Dispatch<React.SetStateAction<number>>;
   suggestionItems: T[];
   setSuggestionItems: React.Dispatch<React.SetStateAction<T[]>>;
+  footerItemIndex: number;
+  setfooterItemIndex: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
why do we need it to be of type  React.Dispatch<React.SetStateAction<number>> ?

Reply:
absolutely no idea, I just copied the setFocusItemIndex type

Reply:
no worries.. i will research more on this but i do its usage in the codebase :)